opm/simulators/linalg/ISTLSolverEbos.hpp

```diff
@@ -150,11 +151,11 @@ namespace Opm
             extractParallelGridInformationToISTL(simulator_.vanguard().grid(), parallelInformation_);
 
             // For some reason simulator_.model().elementMapper() is not initialized at this stage
-            // Hence const auto& elemMapper = simulator_.model().elementMapper(); does not work.
+            const auto& elemMapper = simulator_.model().elementMapper(); //does not work.
             // Set it up manually
```

Comment:
@blattms Can you elaborate on this. Using the elemMapper directly seems to work for me.

Reply:
Ok. It does not work in parallel.